pwdtk/watchers.py

```diff
@@ -29,11 +30,25 @@ def decorated(self, password):
         passwd_str = password if LOG_PASSWORDS else '********'
         logger.debug(
             "potential passwd change detected %r %s",
-            self.username, repr(passwd_str))
+            self.username,
```

Comment:
Just a question. Why %r for self.username? It's always in ASCII, right?

Reply:
You're right, that's just being paranoid:

with %r it's easier to detect funny things, like:

None, a username with leading or trailing spaces an accidentally passed object, etc.